tools/src/main/scala/com/nvidia/spark/rapids/tool/ToolTextFileWriter.scala

```diff
@@ -25,8 +25,7 @@ import org.apache.spark.sql.rapids.tool.profiling._
 /**
  * Class for writing local files, allows writing to distributed file systems.
  */
-class ToolTextFileWriter(finalOutputDir: String, logFileName: String) extends Logging {
-
+case class ToolTextFileWriter(finalOutputDir: String, logFileName: String) extends Logging {
```

Comment:
why this change, please remove

Reply:
I made this change as a shortcut to get the app id  into the label https://github.com/NVIDIA/spark-rapids/pull/2640/files#diff-8dc61637872038b82e8dda72007efa65d71ff93aae90b312c7185abd320dbe87R175

Reply:
Yeah I'd much rather not reuse this and have the app id passed in if we want it there.  CollectInformation can pass it into generateDotGraph